docs/index.rst

```diff
@@ -45,6 +45,7 @@ To build this theme we drew inspiration from other great projects on the web tha
 - Docker: https://docs.docker.com/engine/docker-overview/
 - PyTorch: https://pytorch.org/docs/stable/notes/autograd.html
 
+Thanks to `@drammock <https://github.com/drammock>`_ for initial design of the theme logo.
```

Comment:
maybe this can then be moved to the aknowledgment section

Reply:
isn't it already there? the bullet list just above is the acknowledgements

Reply:
my bad I though I moved to another page but I'm confusing with Gallery